deployer/README.md

```diff
@@ -1175,3 +1175,260 @@ the first created. You can check if the route in question is defined in multiple
     logging     kibana-ops   kibana-ops.example.com              logging-kibana-ops
 
 (In this example there are no overlapping routes.)
+
+## Splitting your Fluentd for high scale capacity (Advanced)
+
+In scenarios where the number of Fluentd nodes starts to exceed 100, the watching
```

Comment:
My thought with saying '100' was that even if they wouldn't hit the default limit with maxRequestsInFlight, they would want to re-architect their logging solution to decrease the load it's putting on the API server in general. Would it just be appropriate to call that information out specifically? (2 watches per Fluentd and confirm the default value)